shimmer/docs/cli-wallet/docs/00_welcome.md

```diff
@@ -0,0 +1,17 @@
+# Welcome
+
+![cli-wallet](/img/cli-wallet.gif)
+
+The [cli-wallet](https://github.com/iotaledger/iota-sdk/tree/develop/cli) is a stateful Command Line Interface wrapper around
+[iota-sdk](https://github.com/iotaledger/iota-sdk).
```

Comment:
Should actually be the sdk folder I think, because in the root there is also the cli
```suggestion
[iota-sdk](https://github.com/iotaledger/iota-sdk/tree/develop/sdk).
```